gensim/utils.py

```diff
@@ -1154,29 +1154,29 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
             return default_res
 
 
-def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
-    """
-    Run command with arguments and return its output as a byte string.
-    Backported from Python 2.7 as it's implemented as pure python on stdlib.
-
-    >>> check_output(args=['/usr/bin/python', '--version'])
-    Python 2.6.2
-    Added extra KeyboardInterrupt handling
+def check_output(args, flag=True):
```

Comment:
It seems like the travis build succeeded after develop was merged into issue. Is the PR good to go?